web/routing.go

```diff
@@ -233,21 +134,19 @@ func SetupAdminRoutes(router *echo.Echo) error {
 		mws = append(mws, middlewares.BasicAuth(config.GetConfig().AdminSecretFileName))
 	}
 
-	instances.Routes(router.Group("/instances", mws...))
 	version.Routes(router.Group("/version", mws...))
-	metrics.Routes(router.Group("/metrics", mws...))
-	realtime.Routes(router.Group("/realtime", mws...))
 
 	setupRecover(router)
 
 	router.HTTPErrorHandler = errors.ErrorHandler
 	return nil
 }
 
-// CreateSubdomainProxy returns a new web server that will handle that apps
+// SetupMajorRoutes returns a new web server that will handle that apps
```

Comment:
There is no more apps

Reply:
Why major routes? There is no minor routes :)

Reply:
Ok for the comment. 

It refers to "major" is server.go. I will refactor this function.